### ISSUE TITLE
Sorting records by Import GUID fails

### ISSUE THREAD

**Issue by claire-halpin** [NERC-CEH/irecord-app#296]
**Describe the issue**
In Explore my records, click on the column heading 'Import GUID' to sort records and I receive the error message 'Elasticsearch query failed'

**To Reproduce**
Steps to reproduce the behaviour:
1. Select Explore my records
2. Configure the list to include the 'Import GUID' attribute
3. Click on the Import GUID column heading to sort the list
4. Error 'Elasticsearch query failed' displayed in pop-up box

![image](https://github.com/user-attachments/assets/1fa76d94-10ab-485b-87ad-1f6e02344160)

**Additional context**
I'm in test mode, checking the spreadsheet import of Bryophyte records

**Comment by claire-halpin** [comment]
Having investigated further, I note that:
- On the Explore records page, using a filter in the Import GUID seems to work
- If I add Import GUID to the Verify - main page, and attempt to sort **or filter** on this attribute, I get the same Elasticsearch query error

**Comment by sacrevert** [comment]
@claire-halpin if there is a remaining error here shouldnt this be left open?

**Comment by claire-halpin** [comment]
Yes it should! Sorry, I'm not familiar with this issue tracker and thought I was just closing the comment I'd made.

**Comment by claire-halpin** [comment]
Thanks Oli. Didn't realise I'd closed the whole issue - have reopened it.
